tpp-sample/src/RedeemStandingOrderSampleTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Tokenio.Proto.Common.AliasProtos;
```

Comment:
Name conflict in java and C# 
1. In java:
RedeemStandingOrderTokenSampleTest.java

in c#
RedeemStandingOrderSampleTest.cs